File: semester_1/praktikum/daspro_3/cek_segitiga.py
def cek_segitiga(a, b, c) :
    if a + b > c and a + c > b and b + c > a :
        if a == b == c :
            return "Sama Sisi"
        elif a == b > c or b == c > a or a == c > b :
            return "Sama Kaki"
        elif a != b or b != c or c != a :
            return "Sembarang"
    else :
        return "Tidak dapat membentuk segitiga"

print (cek_segitiga(1, 1, 1))
print (cek_segitiga(2, 1, 2))
print (cek_segitiga(5, 2, 6))
print (cek_segitiga(3, 4, 5))
print (cek_segitiga(3, 4, 100))

File: semester_2/alpro/2_kasus/CekSegitiga.c
// Nama File 	  : CekSegitiga.c
// Deskripsi 	  : CekSegitiga menentukan jenis sebuah segitiga dari 3 buah bil int
// Pembuat   	  : Ruth Septriana Sipangkar_2406124120024
// Tgl Pembuatan: 02/03/25, 22.24 WIB

// Header
#include <stdio.h>

// Program Utama
int main() {
// Kamus
  int i, j, k;
  
// Algoritma
printf("Masukkan nilai sisi pertama: ");
scanf("%d", &i);
printf("Masukkan nilai sisi kedua: ");
scanf("%d", &j);
printf("Masukkan nilai sisi ketiga: ");
scanf("%d", &k);

// versi 1
  if (i>0 && j>0 && k>0) {
    if (i == j && i == k && j == k) {
        printf("Merupakan segitiga sama sisi");}
    else if (i==j || i==k || j==k) {
        printf("Merupakan segitiga sama kaki");}
    else {
        printf("Merupakan segitiga sembarang");}
  }
  else {
    printf("Terdapat nilai yang bukan sisi segitiga\n");
  }

// versi 2
if (i+j >k || j+k >i || k+i >j) {
  if (i == j && i == k && j == k) {
      printf("Merupakan segitiga sama sisi");}
  else if (i==j || i==k || j==k) {
      printf("Merupakan segitiga sama kaki");}
  else {
      printf("Merupakan segitiga sembarang");}
}
else {
  printf("Terdapat nilai yang bukan sisi segitiga");
}

  return 0;
}